noir-projects/aztec-nr/aztec/src/macros/notes/mod.nr

```diff
@@ -433,17 +434,22 @@ comptime fn generate_setup_payload(
 
     // Then the log plaintext ones
     let log_plaintext_length = indexed_fixed_fields.len() * 32 + 64;
-    let setup_log_plaintext =
+    let setup_log_plaintext: Quoted =
         get_setup_log_plaintext_body(s, log_plaintext_length, indexed_nullable_fields);
 
-    // Then we compute values for `encrypt_log(...)` function
-    let untagged_encrypted_log_byte_length = OVERHEAD_SIZE
+    // Then we compute values for `encrypt_log(...)` function.
+    // First, the length of the items that are broken into bytes:
+    let encrypted_log_bytes_length = 1 /* eph_pk_sign */
+        + 48 /* header_ciphertext */
```

Comment:
Apologies for the hard-coded constants; this partial note stuff will die soon, though.